scripts/migrate_actions_settings_expressions.js

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const pathLib = require('path');
+const program = require('commander');
+
+const stack_version = program.stack || require('../lerna.json').version;
+
+program
+	.version(stack_version)
+	.option('-d, --debug', 'display more info')
+	.option('-q, --quiet', 'do not display logs')
+	.option('-p, --path [value]', 'path of the settings.json to update')
+	.option('-f, --folder [value]', 'path of the folder containain json to update');
+
+program.on('--help', () => {
+	console.log('To update your project settings : ');
+	console.log('>node bin/version.js --path ../yourapp/settings.json');
```

Comment:
update the help with the name of the script migrate_actions_settings_expressions.js